lms/static/sass/course/instructor/_cohort-management.scss

```diff
@@ -0,0 +1,143 @@
+.cohort-management {
+
+  .has-option-unavailable { // Given to <selects> that have some options that are unavailable
+
+    .option-unavailable { // Allows granular styling (i.e., red) for those unavailable options
+      background: $red-l5;
+      color: $error-red;
+    }
+  }
+
+  .has-other { // Given to groups which have an 'other' input that appears when needed
+    display: inline-block;
+
+    label {
+      display: inline-block;
+    }
+
+    .input-group-other {
+      display: none;
+
+      &.is-visible {
+        display: inline;
+        position: relative;
+        overflow: auto;
+        width: 100%;
+        height: auto;
+        margin: 0 0 0 $baseline;
+        padding: inherit;
+        border: inherit;
+        clip: auto;
+      }
+    }
+  }
+
+  .cohort-management-nav {
+    margin-bottom: $baseline;
+
+    .action-create {
+      margin-top: $baseline;
+    }
+  }
+
+  .cohort-management-settings {
+    @include clearfix();
+    background: $gray-l5;
+
+    .cohort-manage-group-header {
+      padding: 0;
+      border-bottom: 0;
+
+      .group-header-title {
+        padding: $baseline;
+        margin-bottom: 0;
+        border-bottom: 1px solid $gray-l3;
+        text-transform: none;
+        letter-spacing: normal;
+      }
+
+      .cohort-management-group-setup {
+        padding: $baseline;
+
+        .cohort-management-group-text {
+          display: inline-block;
+          width: flex-grid(9);
+        }
+
+        .cohort-management-group-actions {
+          display: inline-block;
+          width: flex-grid(3);
+          vertical-align: top;
+
+          .float-right {
+            float: right;
+          }
+        }
+      }
+    }
+
+    .cohort-management-details,
+    .cohort-management-group-add {
+      border-top: ($baseline/5) solid $gray-l4;
+      background: $white;
+
+      .cohort-details-name {
+        @extend %t-action1;
+        display: block;
+        width: 100%;
+        padding: ($baseline/2);
+        margin-bottom: ($baseline*2);
+      }
+
+      .cohort-section-header {
+        margin-top: ($baseline*1.5);
+        padding: $baseline 0 ($baseline/2) 0;
```

Comment:
Do you need to reset horizontal padding to 0 here or can you use ``padding-top`` and ``padding-bottom`` rules?